File: restaurant-backend/src/main/java/com/app/restaurant/service/IngredientService.java
package com.app.restaurant.service;

import com.app.restaurant.exception.ResourceNotFoundException;
import com.app.restaurant.model.Ingredient;
import com.app.restaurant.model.Meal;
import com.app.restaurant.repository.IngredientRepository;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import java.util.List;

@Service
public class IngredientService {
    @Autowired
    IngredientRepository ingredientrepo;



    public List<Ingredient> availableIngredient(){

        return ingredientrepo.findAll();

    }
}

File: restaurant-frontend/src/Component/Services/IngredientService.js
import axios from "axios";

const INGREDIENT_BASE_REST_API_URL= 'http://localhost:8080/api/ingredient';


class IngredientService{
    availableIngredient(){
        return axios.get(INGREDIENT_BASE_REST_API_URL)
    }

  

}

export default new IngredientService();